Wrappers/Python/ccpi/viewer/CILViewer.py

```diff
@@ -414,10 +413,39 @@ def image2world(self, image_coordinates):
         return [(image_coordinates[i]) * spac[i] + orig[i] for i in range(3)]
 
     def GetImageWorldExtent(self):
+        """Deprecated. Use `GetDataExtentInWorld` and `GetVoxelsFromExtent`."""
+        return self.image2world(self.GetInputData().GetExtent()[1::2])
+
+    def GetDataExtentInWorld(self):
         """
-        Compute and return the maximum extent of the image in the rendered world
+        Compute and return the extent of the input data in the rendered world.
         """
-        return self.image2world(self.GetInputData().GetExtent()[1::2])
+        data_extent_image = self.GetInputData().GetExtent()
+        data_extent_world = self.image2worldExtent(data_extent_image)
+        return data_extent_world
+
+    def GetVoxelsFromExtent(self, extent):
```

Comment:
I don't think this method name is really correct. Having thought about it only 1 minute I could suggest something like 
```suggestion
    def GetMinMaxVoxelsFromExtent(self, extent):
```

Reply:
Accepted